src/providers/twitch/IrcMessageHandler.cpp

```diff
@@ -807,10 +807,17 @@ void IrcMessageHandler::handleNoticeMessage(Communi::IrcNoticeMessage *message)
         }
 
         QString tags = message->tags().value("msg-id").toString();
-        if (tags == "bad_delete_message_error" || tags == "usage_delete")
+        if (tags == "usage_delete")
         {
             channel->addMessage(makeSystemMessage(
-                "Usage: /delete <msg-id>. Can't take more than one argument"));
+                "Usage: /delete <msg-id> - Deletes the specified message. "
+                "Can't take more than one argument."));
+        }
+        else if (tags == "bad_delete_message_error")
+        {
+            channel->addMessage(
+                makeSystemMessage("There was a problem deleting the message. "
+                                  "It might be from another channel"));
```

Comment:
This should probably say something about "too old" messages instead/as well? It would be more likely to fail while trying to delete a message in message history, (or an ancient message being replied to as Aiden mentioned to us.)